docs/conf.py

```diff
@@ -70,17 +71,25 @@
 # (note that `group` is the GitHub user or organization)
 issues_github_path = "skops-dev/skops"
 
+REVISION_CMD = "git rev-parse --short HEAD"
 
-def linkcode_resolve(domain, info):
-    if domain != "py":
-        return None
-    if not info["module"]:
+
+def _get_git_revision():
+    try:
+        revision = subprocess.check_output(REVISION_CMD.split()).strip()
+    except (subprocess.CalledProcessError, OSError):
+        print("Failed to execute git to get revision")
         return None
-    filename = info["module"].replace(".", "/")
-    if domain not in ("py", "pyx"):
-        return
```

Comment:
All done! I couldn't edit the changes.rst file in this pr, so I had to create a new one, which is #186

Reply:
> > I couldn't edit the changes.rst file in this pr
> 
> That's strange. What error did you get? Maybe we can figure it out :)

There is no error. The problem is I don't know how to do that :)

Reply:
Hmm, same as with the `conf.py`? You modify the `changes.rst`, then `git add changes.rst`, then `git commit -m "blabla"` and `git push`.

Reply:
Thanks! I use the browser. Is there any way to do that on the browser? Or I'll use the CLI.

Reply:
Hmm, I think it's probably possible in the browser, but I have never used it, so maybe not.

Reply:
I get this error message:
```
 ! [rejected]        main -> main (fetch first)
error: failed to push some refs to 'https://github.com/ayyucedemirbas/skops.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Reply:
Perhaps, doing that through the browser would be the best.

Reply:
Do a `git pull`, git will merge the changes made here to your local branch. That should work without further complications because there should be no merge conflicts. After that, try `git push` again.

Reply:
Yes, I did it (via browser) :) Thanks a lot!